cupyx/scipy/sparse/csr.py

```diff
@@ -698,3 +723,267 @@ def cupy_multiply_by_csr_step2():
         ''',
         'cupy_multiply_by_csr_step2'
     )
+
+
+_BINOPT_MAX_ = '''
+__device__ inline O binopt(T in1, T in2) {
+    return max(in1, in2);
+}
+'''
+
+_BINOPT_MIN_ = '''
+__device__ inline O binopt(T in1, T in2) {
+    return min(in1, in2);
```

Comment:
Why not embed this in the kernel code and just pass a string parameter `min` `max` that is substituted in a template?